protocol/testutil/constants/affiliates.go

```diff
@@ -0,0 +1,32 @@
+package constants
+
+import (
+	affiliates "github.com/dydxprotocol/v4-chain/protocol/x/affiliates/types"
+)
+
+var (
+	DefaultAffiliateTiers = affiliates.AffiliateTiers{
+		Tiers: []affiliates.AffiliateTiers_Tier{
+			{
+				ReqReferredVolume:   0,
+				ReqStakedWholeCoins: 0,
+				TakerFeeSharePpm:    50000,
```

Comment:
Nit: add separators for larger values like `50_000`. Also add comment for humand readable value (5%)